src/lib/shadow-shim-helper-rs/src/shim_event.rs

```diff
@@ -25,6 +25,19 @@ pub struct ShimEventSyscallComplete {
 #[repr(C)]
 pub struct ShimEventAddThreadReq {
     pub ipc_block: ShMemBlockSerialized,
+    // Params for clone syscall
+    pub flags: libc::c_ulong,
+    pub child_stack: UntypedForeignPtr,
+    pub ptid: UntypedForeignPtr,
+    pub ctid: UntypedForeignPtr,
```

Comment:
nit: it should be obvious based on the context, but maybe doc-comment which address space the pointers are valid in since they're in shared memory.